repository-data/webfiles/src/main/resources/site/css/demo.css

```diff
@@ -0,0 +1,98 @@
+body {
```

Comment:
Unless there is a specific reason to call this file `demo.css` - how about renaming it to something that doesn't suggest that a demo/test/experimental file has sneaked into production code? `cw-main.css` or `cw-styles.css` come to mind.

Reply:
that was by-design. This file should be removed as soon as Avi lands in the team :)